e2e/components/Tabs/Tabs-test.avt.e2e.js

```diff
@@ -0,0 +1,215 @@
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('Tabs-contained');
+  });
+
+  test('accessibility-checker tabs contained full width', async ({ page }) => {
+    await visitStory(page, {
+      component: 'Tabs',
+      id: 'components-tabs--contained-full-width',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('Tabs--contained-full-width');
+  });
+
+  test('accessibility-checker tabs contained with icons', async ({ page }) => {
+    await visitStory(page, {
+      component: 'Tabs',
+      id: 'components-tabs--contained-with-icons',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('Tabs-contained-with-icons');
+  });
+
+  test('accessibility-checker tabs contained with secondary labels', async ({
+    page,
+  }) => {
+    await visitStory(page, {
+      component: 'Slider',
+      id: 'components-tabs--contained-with-secondary-labels',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations(
+      'Tabs-contained-with-secondary-labels'
+    );
+  });
+
+  test('accessibility-checker tabs contained with secondary labels and icons', async ({
+    page,
+  }) => {
+    await visitStory(page, {
+      component: 'Tabs',
+      id: 'components-tabs--contained-with-secondary-labels-and-icons',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations(
+      'Tabs--contained-with-secondary-labels-and-icons'
+    );
+  });
+
+  test('accessibility-checker tabs dismissable', async ({ page }) => {
+    await visitStory(page, {
+      component: 'Tabs',
+      id: 'components-tabs--dismissable',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('Tabs--dismissable');
+  });
+
+  test('accessibility-checker tabs dismissable with icons', async ({
+    page,
+  }) => {
+    await visitStory(page, {
+      component: 'Tabs',
+      id: 'components-tabs--dismissable-with-icons',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('Tabs--dismissable-with-icons');
+  });
+
+  test('accessibility-checker tabs icon 20 only', async ({ page }) => {
+    await visitStory(page, {
+      component: 'Tabs',
+      id: 'components-tabs--icon-20-only',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('Tabs--icon-20-only');
+  });
+
+  test('accessibility-checker tabs icon only', async ({ page }) => {
+    await visitStory(page, {
+      component: 'Tabs',
+      id: 'components-tabs--icon-only',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('Tabs--icon-only');
+  });
+
+  test('accessibility-checker tabs manual', async ({ page }) => {
```

Comment:
I don't think we need this a11y test, but we should test the difference between `manual` and `automatic` in the keyboard testing section.